apps/client/src/utils/MatamoConfig.js

```diff
@@ -0,0 +1,41 @@
+import { isProduction } from "../config";
+import { useMatomo } from "@datapunt/matomo-tracker-react";
+import { useContext } from "react";
+import Context from "../context";
+
+const matomoSiteId = {
+  production: 29,
+  acceptence: 37,
+};
+
+export const isProductionAndNotOurs =
+  isProduction && !localStorage.getItem("enterFromDev");
```

Comment:
Please enable tracking from localhost as well. Can this work?
```
!isProduction || ( isProduction && !localStorage.getItem("enterFromDev") )
```

Or turn it around: only set the `enterFromDev` on production